hamilton/driver.py

```diff
@@ -417,8 +418,15 @@ def __init__(
             adapter.call_all_lifecycle_hooks_sync("pre_do_anything")
         error = None
         self.graph_modules = modules
+        # Feels like this should be public class attribute.
+        self.allow_module_overrides = allow_module_overrides
```

Comment:
Start non-public IMO, then we can make it public. That said, we're not doing a good job here of differentiating, so just don't worry about it.